smart-contract-verifier/src/compiler/compilers.rs

```diff
@@ -98,14 +98,14 @@ mod tests {
     use std::{env::temp_dir, str::FromStr};
 
     use crate::consts::DEFAULT_COMPILER_LIST;
-    use async_once_cell::OnceCell;
     use ethers_solc::artifacts::{Source, Sources};
     use std::default::Default;
+    use tokio::sync::OnceCell;
```

Comment:
What was the reason behind changing `async_once_cell::OnceCell` on `tokio::sync::OnceCell`?

Reply:
why include some questionable additional dependency, when we already have it in tokio?

Reply:
Okay, so just because the second library is more authoritative